gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Thu Oct 10 13:23:52 PDT 2019
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.2-all.zip
```

Comment:
Why is this change needed? *-all instead of *-bin.

Reply:
This is what ./gradlew build did.  Not sure why.   Reverting it is probably a fruitless endeavor.